leet_69.py

```diff
@@ -0,0 +1,7 @@
+import math
+class Solution:
+    def mySqrt() -> int:
+        x = int(input("Please enter an integer:"))
+        rt = int(math.sqrt(x))
+        print(rt)
```

Comment:
Also in original signature, return type is int, so instead of printing, you must return an integer. Leetcode treats printing as debugging mode.